glide.yaml

```diff
@@ -0,0 +1,26 @@
+package: github.com/openshift/must-gather
+import:
+- package: github.com/spf13/cobra
+  repo:    git@github.com:spf13/cobra
+  version: 7547e83b2d85fd1893c7d76916f67689d761fecb
+- package: github.com/spf13/pflag
+  repo:    git@github.com:spf13/pflag
+  version: 24fa6976df40757dce6aea913e7b81ade90530e1
+- package: k8s.io/client-go
```

Comment:
pin to a version similarly to what we do in origin, iow. `kubernetes-1.11.1` for now, or you can even go with `kubernetes-1.12.4` which is what we'll get with the rebase